conan/cli/commands/inspect.py

```diff
@@ -1,21 +1,20 @@
-import inspect as python_inspect
 import os
 
 from conan.api.output import cli_out_write
-from conan.cli.command import conan_command
+from conan.cli.command import conan_command, OnceArgument
 from conan.cli.formatters import default_json_formatter
 
 
 def inspect_text_formatter(data):
-    for name, value in data.items():
+    for name, value in sorted(data.items(), key=lambda x: x[0]):
```

Comment:
why this lambda? Not very intuitive

Reply:
While playing with the output I realized it would be insertion-order dependant and implemented a sort on the items to have them show up alphabetically. Maybe we want to have a custom order and just have to be carefurl about when we insert the result in the serialization?

Reply:
Still not sure. ``sorted(mydict.items())`` already sorts by the first element (key), no need for the lambda